runtime/Cpp/runtime/src/TokenStreamRewriter.cpp

```diff
@@ -251,7 +251,7 @@ std::string TokenStreamRewriter::getText(const Interval &interval) {
 }
 
 std::string TokenStreamRewriter::getText(const std::string &programName, const Interval &interval) {
-  std::vector<TokenStreamRewriter::RewriteOperation*> rewrites = _programs[programName];
+  std::vector<TokenStreamRewriter::RewriteOperation*> *rewrites = &_programs[programName];
```

Comment:
Good find! However, I'd prefer to use a reference here, instead of a pointer, e.g.

`std::vector<TokenStreamRewriter::RewriteOperation*> &rewrites = _programs[programName];`